src/components/TodoApp/index.jsx

```diff
@@ -0,0 +1,24 @@
+import AddTodoForm from './AddTodoForm';
+import Header from './Header';
+import List from './List';
+import useTodos from './useTodos';
```

Comment:
```suggestion
import AddTodoForm from './AddTodoForm';
import Header from './Header';
import List from './List';

import useTodos from './useTodos';
```

의미상으로 구분되는 경우 빈 줄을 넣어 구분해주면 코드가독성이 좋아질 것 같아요.
컴포넌트와 훅으로 나눠줘도 괜찮겠네요 ;)